spec/index.html

```diff
@@ -423,5 +440,98 @@ <h3>Examples</h3>
     </section>
   </section>
 
+    <section id="best-practices" class="informative">
+      <h2>Best Practices</h2>
+
+      <p>Here, we propose to YAML-LD users a bit of advice which, although optional, might suggest one or two
+        useful thoughts.</p>
+
+       <div class="practice">
+          <p class="practicedesc">
+            <span id="use-json-ld-best-practices" class="practicelab">Follow JSON-LD best practices</span>
+            …in order to achieve a greater level of reusability, performance, and human friendliness of YAML-LD aware
+            systems. The [[json-ld-bp]] document is relevant to YAML-LD no less than it is to [[JSON-LD]].
+          </p>
+       </div>
+
+      <div class="practice">
+        <p class="practicedesc">
+          <span id="prebuilt-contexts" class="practicelab">Do not force users to author contexts</span>
+
+          Instead, provide pre-built contexts that the user can reference by URL for majority of common use cases.
+        </p>
+      </div>
+
+      <p>YAML-LD is intended to simplify authoring of Linked Data for a wide range of domain experts: its target
+        audience is not comprised solely of IT professionals. YAML is chosen as a medium to minimize syntactic noise,
+        to keep the authored documents concise and clear. [[JSON-LD]] (and hence YAML-LD) Context comprises a special
+        language of its own. A requirement to <i>author</i> such a context would make the domain expert's job much
+        harder, — which we, as system architects and developers, should try to avoid.</p>
```

Comment:
```suggestion
      <p>YAML-LD is intended to simplify the authoring of Linked Data for a wide range of domain experts; its target
        audience is not comprised solely of IT professionals. YAML is chosen as a medium to minimize syntactic noise,
        and to keep the authored documents concise and clear. [[JSON-LD]] (and hence YAML-LD) Context comprises a special
        language of its own. A requirement to <i>author</i> such a context would make the domain expert's job much
        harder &mdash; which we, as system architects and developers, should try to avoid.</p>
```